person.rb

```diff
@@ -1,22 +1,34 @@
 # Person class
-class Person
-  attr_reader :id
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_reader :id, :rental
   attr_accessor :name, :age
```

Comment:
Please add Getters and Setters (attr_accessor) for `@rental`, not only Getters attr_reader as you did because you suppose to modify it not only read it.